code/_globalvars/traits/_traits.dm

```diff
@@ -462,6 +462,7 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_THINKING_IN_CHARACTER" = TRAIT_THINKING_IN_CHARACTER,
 		"TRAIT_THROWINGARM" = TRAIT_THROWINGARM,
 		"TRAIT_TIME_STOP_IMMUNE" = TRAIT_TIME_STOP_IMMUNE,
+		"TRAIT_TOO_TALL" = TRAIT_TOO_TALL,
```

Comment:
since this is human only you can put it in the `/human` category of traits if u want

all the other traits aren't particularly good at following that

Reply:
i literally cant find this 'human' category anywhere

Reply:
you make it, like these
![image](https://github.com/tgstation/tgstation/assets/51863163/c2a20a23-38ad-4d62-8ae3-601c2f879cbe)